x-pack/plugins/maps/public/shared/layers/sources/es_geo_grid_source/es_geo_grid_source.js

```diff
@@ -172,7 +201,9 @@ export class ESGeoGridSource extends AbstractESSource {
     const searchSource  = await this._makeSearchSource(searchFilters, 0);
     const aggConfigs = new AggConfigs(indexPattern, this._makeAggConfigs(searchFilters.geogridPrecision), aggSchemas.all);
     searchSource.setField('aggs', aggConfigs.toDsl());
-    const esResponse = await this._runEsQuery(layerName, searchSource, 'Elasticsearch geohash_grid aggregation request');
+    const esResponse = await this._runEsQuery(layerName, searchSource, i18n.translate('xpack.maps.source.esGrid.inspectorDescription', {
```

Comment:
default message has 2 sets of quotations around it. Remove ticks. Also, lets change the string to `Elasticsearch geo grid aggregation request` since we now use geotile_grid and really the aggregation name is not needed